packages/studio/src/components/common/ScopeFilterField.tsx

```diff
@@ -0,0 +1,63 @@
+import { Tooltip } from "react-tooltip";
+import PillPickerInput from "../PillPicker/PillPickerInput";
+import { streamScopeFormData } from "../PageSettingsButton/EntityPageModal";
+import { StreamScope } from "@yext/studio-plugin";
+import { useCallback } from "react";
+
+// TODO (SLAP-2918): Combine this with streamScopeFormData once this component
+// is used for page settings
+const emptyTextData: { [field in keyof StreamScope]: string } = {
+  entityIds: "No entities found in the account.",
+  entityTypes: "No entity types found in the account.",
+  savedFilterIds: "No saved filters found in the account.",
+};
+
+export interface ScopeFilterFieldProps {
+  field: string;
+  allItems: {
```

Comment:
Is there a better name than allItems? I was tracing how we pass this down and there's a lot of layers, and hopefully the name could be more helpful when doing this

Reply:
Renamed to `filterOptions`